### PR TITLE
fix contribution.md

### PR DESCRIPTION
**Pull request by khushi818**
Issue : (#105)

### Fix 
- Updated the required steps necessary to set up discord-slash and rds-backend with discord

close(#105)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -76,34 +76,63 @@ Next you will have to set up the wrangler cli, so that you can connect to your c
...
 
-Now add the public key in `rds-backend`
+Now to setup discord in `rds-backend`
+
+- Go to Your cloudflare `dashboard > workers > discord-slash-commands > settings > variables > edit Variables`
+- Now add one more variables to your environment:
 
-- Go to `local.js` in config
-- Create following there
+  - `RDS_SEVERLESS_PUBLIC_KEY` (`generate key from the link given above`)
```
Comment (bharati-21) [inline CONTRIBUTING.md:96]: Which `generate key from the link given above`? This is not very clear.

Comment (bharati-21) [inline CONTRIBUTING.md:96]: Also is it SERVERLESS or SEVERLESS? Please ensure no typos are present.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -76,34 +76,63 @@ Next you will have to set up the wrangler cli, so that you can connect to your c
...
-  botPublicKey:<Public key generated in the format similar to development.js>
-}
+ services:{
+   discordBot: {
+     baseUrl: "<Cloudflare_url>",
+   },
+ }
+
+ botToken: {
+   botPublicKey: "<Public key generated in the format similar to development.js>",
```
Comment (bharati-21) [inline CONTRIBUTING.md:108]: What does similar to `development.js` mean?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -76,34 +76,63 @@ Next you will have to set up the wrangler cli, so that you can connect to your c
...
+ },
+
+
+ rdsServerlessBot: {
+   rdsServerLessPrivateKey: "<RDS_SERVERLESS_PRIVATE_KEY in the format similar to development.js>",
```
Comment (bharati-21) [inline CONTRIBUTING.md:113]: Again the same thing.

Comment (bharati-21) [reply CONTRIBUTING.md:113]: No where above development.js is mentioned.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -76,34 +76,63 @@ Next you will have to set up the wrangler cli, so that you can connect to your c
...
-  botPublicKey:<Public key generated in the format similar to development.js>
-}
+ services:{
+   discordBot: {
+     baseUrl: "<Cloudflare_url>",
+   },
+ }
+
+ botToken: {
+   botPublicKey: "<Public key generated in the format similar to development.js in config>",
```
Comment (bharati-21) [inline CONTRIBUTING.md:108]: What I meant was `similar to development.js` is misleading. We never edited the development.js file since the beginning.

**Comment by sahsisunny** [comment]
> Issue : (#105) Fix - Updated the required steps necessary to set up discord-slash and rds-backend with discord
> 
> close(#105)

https://www.markdownguide.org/basic-syntax/ Please read in free time

**Comment by khushi818** [comment]
> > Issue : (#105) Fix - Updated the required steps necessary to set up discord-slash and rds-backend with discord
> > close(#105)
> 
> Why aren't you using Markdown formatting skills for the PR description? Check out https://www.markdownguide.org/basic-syntax/ when you have free time.

got it

File: CONTRIBUTING.md (hunk 4 above)
Comment (khushi818) [reply CONTRIBUTING.md:108]: That s how it was mention earlier , and it is not misleading as we are talking about format. And it is clearly mentioned that we have change in local.js

Comment (bharati-21) [reply CONTRIBUTING.md:108]: So why not change it to `local.js`? Why keep it `development.js`?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -76,34 +76,63 @@ Next you will have to set up the wrangler cli, so that you can connect to your c
...
+ },
+
+
+ rdsServerlessBot: {
+   rdsServerLessPrivateKey: "<RDS_SERVERLESS_PRIVATE_KEY in the format similar to development.js in config>",
+   ttl: 60,
```
Comment (bharati-21) [inline CONTRIBUTING.md:114]: Same here. What is the use of this?

File: CONTRIBUTING.md (hunk 4 above)
Comment (khushi818) [reply CONTRIBUTING.md:108]: we can't take reference from local.js

File: CONTRIBUTING.md (hunk 5 above)
Comment (khushi818) [reply CONTRIBUTING.md:114]: formatting for the keys

**Review by bharati-21 — APPROVED** [review]
LGTM!

**Review by kotesh-arya — APPROVED** [review]

**Pull request merged**